### DISCUSSION TITLE
Support for DotNet

### DISCUSSION THREAD

**Discussion by Seneti42** [koliveira15/nx-sonarqube#52] [Q&A] [answered]
Hi,

I currently have a monorepo with Angular and DotNet projects, I know this plugin will support scanning the Angular code, will it work with DotNet applications as well? Given the different nature required by Sonar to prepare a scan, build the application then analyse/publish the results.

**Comment by koliveira15** [comment, marked as answer]
> Hi,
> 
> I currently have a monorepo with Angular and DotNet projects, I know this plugin will support scanning the Angular code, will it work with DotNet applications as well? Given the different nature required by Sonar to prepare a scan, build the application then analyse/publish the results.

Hi @Seneti42,

It will not work .NET applications as of right now because the executor specifically looks at the test target's **Jest** configuration and respective coverageDirectory path so it can upload test coverage results. I'm open to expanding the allowed test runner list (vite & whatever .net uses) if folks want to contribute back to the plugin.
